iothub_client/CMakeLists.txt

```diff
@@ -40,11 +40,11 @@ set(iothub_client_h_files
     ./inc/iothub_client_version.h
     ./inc/iothub_device_client.h
     ./inc/iothub_device_client_ll.h
+    ./inc/iothub_message.h
+    ./inc/internal/iothub_message_private.h
     ./inc/iothub_module_client.h
     ./inc/iothub_module_client_ll.h
     ./inc/iothub_transport_ll.h
-    ./inc/iothub_message.h
-    ./inc/internal/iothub_message_private.h
     ./inc/internal/iothubtransport.h
 )
```

Comment:
nit: why change things that don't need to be changed? or am I missing something here

Reply:
As I went though, made items alphabetical.  Although this could be argues its still not.  I can take it out if unwanted.

Reply:
yea it's more of a nitpick to keep PR's as small as possible. Especially if there's a new feature. Helps to keep the focus from a reviewing side on the new feature and not on unrelated changes. Plus PR's are less daunting with least amount of change. A separate PR who's purpose is to clean/patch things up like this bundles things for a more pointed review.